isis/src/lro/apps/lrowaccal/main.cpp

```diff
@@ -231,7 +231,7 @@ void IsisMain () {
     else {
       responsivity = radPvl["Radiance"];
       for (int i = 0; i < bands.size(); i++)
-        g_radianceResponsivity.push_back(toDouble(responsivity[toInt(bands[i]) - 1]) / 1000.0);
+        g_radianceResponsivity.push_back(toDouble(responsivity[toInt(bands[i]) - 1]) * 1000.0);
```

Comment:
I see initially you set the value, which was originally in nano meters I believe, to be represented in micro meters (*.001) but then you went the other direction. Were the original values not in nanometers?